pygments/lexers/thingsdb.py

```diff
@@ -39,21 +39,24 @@ class ThingsDBLexer(RegexLexer):
             (r'([-+]?0x[0-9a-fA-F]+)', Number.Hex),
             (r'[-+]?[0-9]+', Number.Integer),
             (r'[-+]?((inf|nan)([^0-9A-Za-z_]|$)|[0-9]*\.[0-9]+(e[+-][0-9]+)?)',
-            Number.Float),
+             Number.Float),
 
             # strings
             (r'(?:"(?:[^"]*)")+', String.Double),
             (r"(?:'(?:[^']*)')+", String.Single),
+            (r"(?:`(?:[^`]*)`)+", String.Backtick),
 
             # literals
             (r'(true|false|nil)\b', Keyword.Constant),
 
+            # name constants
+            (r'(FULL|USER|GRANT|CHANGE|JOIN|RUN|QUERY|'
+             r'DEBUG|INFO|WARNING|ERROR|CRITICAL|'
+             r'NO_IDS|INT_MIN|INT_MAX)\b', Name.Constant),
```

Comment:
```suggestion
            (words(('FULL', 'USER', 'GRANT', 'CHANGE', 'JOIN', 'RUN',
                    'QUERY', 'DEBUG', 'INFO', 'WARNING', 'ERROR', 'CRITICAL',
                    'NO_IDS', 'INT_MIN', 'INT_MAX'), suffix=r'\b'),
             Name.Constant),
```

Reply:
Oh well. I was going to change such regexes to `words()` below to, but testing it locally, it appears that `words()` introduces capturing groups (they should probably be non-capturing) which makes it not usable with `bygroups()` later. I'll keep these as big regexes for now, will take a look later at whether we can make `words()`'s introduced groups non-capturing.